tests/reg_tests/test_solver_combos.py

```diff
@@ -83,6 +83,33 @@
             "dissContMagnitude": 0.1,
         },
     },
+    # RANS test with AGMG preconditioner
```

Comment:
The solver was only broken for AGMGLevels > 1, which is why this test was working. This test will not work with AGMGLevels > 1 though because the mesh is too coarse. AGMG should work the same now compared to the main branch as I mentioned in the other comment.